### ISSUE TITLE
bad result on kitti

### ISSUE THREAD

**Issue by whitesockcat** [princeton-vl/DPVO#4]
![image](https://user-images.githubusercontent.com/18005530/184280603-e4c17907-1cec-43d2-88f8-1b277cd3700a.png)
![image](https://user-images.githubusercontent.com/18005530/184280668-9bc42b92-0d43-4982-b05c-9d5f090417a9.png)

is there any way to solve this scale problem?

**Comment by zachteed** [comment]
Yeah that doesn’t look good, thanks for bringing this up. It looks like there is some bias in the driving scenarios causing the scale to constantly expand.

I will look into this and reply to this thread in a few days. There might be a simple fix, maybe not. The hacky solution would be to impose a prior on depth of the road immediately in front of the car but that’s not ideal

**Comment by whitesockcat** [comment]
Thx😀, looking forward to getting your update soon

**Comment by aniket-gupta1** [comment]
@whitesockcat what stride did you use for running on kitti?

**Comment by whitesockcat** [comment]
> @whitesockcat what stride did you use for running on kitti?
1

**Comment by RaymonHE** [comment]
> ![image](https://user-images.githubusercontent.com/18005530/184280603-e4c17907-1cec-43d2-88f8-1b277cd3700a.png) ![image](https://user-images.githubusercontent.com/18005530/184280668-9bc42b92-0d43-4982-b05c-9d5f090417a9.png)
> 
> is there any way to solve this scale problem?

Hi @whitesockcat , thanks for sharing the result on KITTI. I am new to KITTI and neural network. I wonder how you test this algorithm on KITTI datasets. Do you just use KITTI as the input and implement the demo.py as shown in the README file, the same way as TartanAir, like `python demo.py --imagedir=<path to image_left> --calib=calib/tartan.txt --stride=1 --viz `, what's the format and how you set the calib and stride file? I would be very grateful if you can give me some suggestion.

**Comment by nnop** [comment]
Any fix on this issue?
